### ISSUE TITLE
No wasm package

### ISSUE THREAD

**Issue by wintersteiger** [project-everest/everest#73]
Not sure what currently is the recommended branch for the `everest` script, but when I just tried to use `master`, it complained about the wasm package not being available:

```
...$ ./everest check
# Switching to the everest directory ... now in /data/cwinter/everest
...
# All required OS packages found.
[ERROR] No package matches wasm.1.0.1.
```
```
...$ git branch -vv
* master 5cefdf1 [origin/master] [CI] automatic upgrade
```
```
...$ ocaml --version
The OCaml toplevel, version 4.05.0
```

**Comment by msprotz** [comment]
You may need to switch to a more recent opam. Can you check if you're on OPAM 1.X? If so, I recommend deleting your $(which opam) and re-running everest check.

**Comment by wintersteiger** [comment]
`1.2.2` - came with Ubuntu 18.04. Once I remove that (package), `everest check` just complains that there isn't one:
```
$ ./everest check
# Switching to the everest directory ... now in /data/cwinter/everest

Checking environment
... which found
... github.com access ok
... ocaml minimum version requirements met
ERROR: opam not found
```
 I guess I have to install that manually then?

**Comment by msprotz** [comment]
ah, yes, everest check only installs opam for you on windows; for linux, I think it's merely a matter of adding Anil's ppa and then installing opam from there

**Comment by wintersteiger** [comment]
There's not even an official PPA for opam?

**Comment by wintersteiger** [comment]
Ah, no it is at least semi-official :)

**Comment by wintersteiger** [comment]
Right, that fixed it, thanks! Would be nice if the script would check the opam version, even if just to report it without offering to upgrade automatically.
